src/playBullsAndCows.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable no-console */
+'use strict';
+
+const readline = require('readline');
+const { countBullsAndCows } = require('./countBullsAndCows');
+const { isGuessValid } = require('./isGuessValid');
+
+const terminal = readline.createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+console.log('Welcome to Bulls and Cows!');
+
+const gameResponse = {
+  error: 'Please enter a valid 4-digit number.',
+  victory: 'Congratulations! You guessed the number',
+};
+
+function playBullsAndCows(secretNumber, message) {
+  terminal.question(message, (guess) => {
+    if (!isGuessValid(guess)) {
+      playBullsAndCows(secretNumber, gameResponse.error);
+
+      return;
+    }
+
+    const { bulls, cows } = countBullsAndCows(secretNumber, guess);
+
+    console.log(`Bulls: ${bulls}, Cows: ${cows}. Try again: `);
```

Comment:
Wait, should you write "try again" if it's a victory?

Reply:
victory is when you guess all 4 digits, nope?